webapp/test/App.spec.js

```diff
@@ -0,0 +1,24 @@
+import { mount } from '@vue/test-utils';
+import app from '../src/App.vue';
+
+describe('App.vue', () => {
+
+  let wrapper = mount(app);
+  beforeEach(() => {
+    wrapper = mount(app);
+  });
+
+  it('renders the todo app', () =>{
+    expect(wrapper.contains('div')).toBe(true);
+  });
+
+  describe('when the toggleEditModeb function is called', () => {
+    //TODO reimplement with backend mockup
```

Comment:
You can mock `$apollo` here. https://vue-test-utils.vuejs.org/api/options.html#mocks